extensions/Particle3D/src/Particle3D/PU/PUMaterialManager.cpp

```diff
@@ -187,7 +187,7 @@ bool PUMaterialCache::loadMaterialsFromSearchPaths(std::string_view fileFolder)
 
     if (!(d = opendir(fileFolder.data())))
     {
-        AXLOG("error opendir %s!!!\n", fileFolder.data());
+        AXLOGD("error opendir {}!!!\n", fileFolder.data());
```

Comment:
.data() can be remove

Reply:
I think its easier make for each kind of %... an own PR

Reply:
> I think its easier make for each kind of %... an own PR

If you mean create a new PR per change related to this, then please don't do that. They should all go in this existing PR, and new PRs created will just spam everyone that has notifications turned on for this repo.

Reply:
> > I think its easier make for each kind of %... an own PR
> 
> If you mean create a new PR per change related to this, then please don't do that. They should all go in this existing PR, and new PRs created will just spam everyone that has notifications turned on for this repo.

agree

Reply:
@halx99 
Any hint:
    AXLOGD("Add sprite %0.2f x %02.f", p.x, p.y);
    AXLOGD("Add sprite {} x {}.f", p.x, p.y);  <<<< ?

Reply:
I think was forgotten?
![image](https://github.com/axmolengine/axmol/assets/8652787/d5da122e-005b-4606-be71-e6bf216d91a4)
![image](https://github.com/axmolengine/axmol/assets/8652787/52bb0799-c26c-4bb7-8e43-3856efc7e948)
...some more...

Reply:
> @halx99 Any hint: AXLOGD("Add sprite %0.2f x %02.f", p.x, p.y); AXLOGD("Add sprite {} x {}.f", p.x, p.y); <<<< ?

`AXLOGD("Add sprite {:.2f} x {:.2f}", p.x, p.y);`

Reply:
Another issue:
   `     AXLOGD("downloading... {}%%", percent);`
        
        I think correct is:
`                AXLOGD("downloading... {}%", percent);`